### ISSUE TITLE
bur6  reannotate based on human

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#2309]
- [ ] bur6 | SPAC17G8.03c | transcription regulator complex subunit Bur6 (predicted)
Inconsistent https://www.uniprot.org/uniprot/Q14919
https://github.com/geneontology/go-annotation/issues/2249

**Comment by ValWood** [comment]
https://www.yeastgenome.org/locus/YER159C

I want to check the annotation of BUR6.
At SGD this is annotated as a corepressor and coactivator, 

but isn't this for things which bind to sequence-specific DNA binding transcription factors.

BUR6 appears to bind to
"binds to TBP and can repress transcription by preventing preinitiation complex assembly or stimulate activated transcription"
which would make it a "general transcription factor?"

@pgaudet do you agree?

**Comment by pgaudet** [comment]
I will pass this one to @colinlog

**Comment by colinlog** [comment]
This is a special case. TBP can be said to bind to a specific sequence (TATA). However it is also the transcription start site selection factor that works for all promoters for RNA polymerases 1, 2 and 3. Therefore, TBP is a GTF. That makes TBP's regulators co-activators/ co-repressors of a GTF. The funny thing is: for one promoter, inhibiting it's association with TBP should be called 'repression'. However, NC2's function is in fact to mobilise TBP for the other binding sites in _cis_ via DNA sliding and to allow TBP to bind to other promoters on other chromosomes via 'unbinding of DNA'. Thus NC2/BTAFI 'indirectly' help to activate many genes. 

In conclusion, I agree with Val, GTF is the MF parent for the annotation for BUR6 and BUR6 is a regulator of a GTF, namely TBP.
Hope this helps.

**Comment by ValWood** [comment]
Thanks Colin,

So we don't have a suitable MF term to describe this at present?

https://www.ebi.ac.uk/QuickGO/term/GO:0140110

**Comment by ValWood** [comment]
We need
general transcription initiation factor inhibitor activity ?

**Comment by ValWood** [comment]
We discussed this at the transcription workshop.
Did we decide to annotate to
general transcription initiation factor
and not to create
general transcription initiation factor inhibitor activity ?

@pgaudet @colinlog I can't remember?

**Comment by ValWood** [comment]
ignore, it's the same as
https://github.com/pombase/curation/issues/2707
I'll use GTIF
